dev/app.js

```diff
@@ -75,6 +75,13 @@ search.addWidget(
 );
 
 search.addWidget(
+  instantsearch.widgets.clearAll({
+    container: '#clear-all',
+    autoHideContainer: false
```

Comment:
I would go for a more explicit option here (public facing API). Something like `hideIfNoRefinements`, while still keeping and internal call to the `autoHideContainer` decorator.